docs/core/proto-docs.md

```diff
@@ -9246,12 +9246,15 @@ Params defines the parameters for the collection module.
 Deprecated: use TokenClass
 
 TokenType defines the information of token type.
+It represents a NFTClass whose class_id is token_type.
+
+Note: There is no TokenType instance for FTClass.
 
 
 | Field | Type | Label | Description |
 | ----- | ---- | ----- | ----------- |
 | `contract_id` | [string](#string) |  | contract id associated with the contract. |
-| `token_type` | [string](#string) |  | token type defines the unique identifier of the token type. |
+| `token_type` | [string](#string) |  | token type defines the unique identifier of the token type. the format of the value is identical to that of class_id. |
```

Comment:
My fault. It seems #866 caused this. I thought it was strange, because the proto scripts changed nothing in my environment, even though I changed the protos.

Anyway, we should add CI on this proto relevant changes.

Reply:
Yes, that's a good idea.